### DISCUSSION TITLE
Magic Links Support

### DISCUSSION THREAD

**Discussion by cupcakearmy** [directus/directus#8288] [Feature Requests]
Are there any plans for supporting magic links for signing in?
Making passwords redundant if chosen.

I did not found an issue/discussion for that, if there alrady is sry :)

**Comment by rijkvanzanten** [comment]
I think I know what you're referring to, but just to confirm:

You mean one of those that sends an email that then lets you login?

@aidenfoxx and I are knee-deep in figuring out how to make the auth integrations more modular so we can start implementing things like this 🙂

**Reply by benhaynes** [reply]
Pretty sure that is the request... and I agree with it! 👍

**Reply by cupcakearmy** [reply]
Yeah, email links that send 1 time codes that can be exchanged for authentication. 
Awesome to hear that. If you manage to break it down and make it modular I could help with the "magic link" stuff :)

**Reply by cupcakearmy** [reply]
Should I make an issue?

**Reply by rijkvanzanten** [reply]
Lets keep this feature request open for a bit first so we can think through the exact implementation details and flow 🙂

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
